java/README.md~

```diff
@@ -0,0 +1,9 @@
+Java Rendler Framework:
```

Comment:
Can you remove it? If you then squash the commits, it won't show up.
See for reference: https://github.com/ginatrapani/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit